server/tools/client_credentials_admin.py

```diff
@@ -0,0 +1,179 @@
+# Copyright (C) 2022 Canonical
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+This script administers client credentials in a MongoDB database. Database login
+information must be specified with environment variables. The script prompts user
+to create a new entry, or edit/delete an existing entry. The user will then be prompted to enter new max_priority and allowed_queues data.
+
+"""
+
+import sys
+import json
+import bcrypt
```

Comment:
we should go ahead and add this as a requirement on the server side since it's needed for this